configs/test_iso_AWS.yaml

```diff
@@ -0,0 +1,90 @@
+ROOT: data
+
+DATA_SCENARIOS:
+  test_data:
+    batch_key: batch
+    label_key: celltype
+    organism: mouse
+    assay: expression
+    file: data/adata_norm.h5ad
+  test_data_rep:
+    batch_key: batch
+    label_key: celltype
+    organism: mouse
+    assay: expression
+    file: data/adata_norm.h5ad
```

Comment:
Could we move DATA_SCENARIOS to the bottom please? Looks a bit better there imho.

Reply:
6640afb